### PR TITLE
set up format and style for index HTML page

### PR DESCRIPTION
**Pull request by Leci1259**
Completed :
- header
- nav bar
- modal
- 4 cards

Still need:
- fill in nav bar links
- add replace placeholder images

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>Server Side APIs</title>
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Fredoka+One&family=Lato:ital,wght@1,700&family=Montserrat:wght@600&display=swap" rel="stylesheet">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.5.0/css/bootstrap.min.css">
+        <link rel="stylesheet" href="./assets/css/style.css" />
```
Comment (brouiller) [inline index.html:13]: your folders have capital letters, this filepath has lowercase letters, it will give an error

File: index.html [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+                </div>
+             </section>
+              
+              
+        </main>
+    </body>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.slim.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.1/umd/popper.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.5.0/js/bootstrap.min.js"></script> 
+    <script src="./Assets/JS/script.js"></script>
```
Comment (brouiller) [inline index.html:101]: If you change the file and folders to lowercase, this src attribute will need to be updated

**Review by brouiller — COMMENTED** [review]
Please use lowercase characters for the files and folders.
Recently, Jung recommended using index for the file name of each type. Let's change script.js to index.js 
The css and html look good to me!

**Review by DeeScrump — APPROVED** [review]
I viewed these in github, agree with Bradley's comments.  Else it looks good.

**Pull request merged**